### PR TITLE
Rovert2

### PR DESCRIPTION
**Pull request by Rovert64**
PRPR (Peach Ruins Pull Request)
sm64 beyond the penis (it's done)

### CODE REVIEW

File: src/game/texscroll.c [hunk 1]
```diff
@@ -117,4 +117,8 @@ void scroll_textures() {
 		scroll_textures_pss();
 	}
 
+	if(SCROLL_CONDITION(sSegmentROMTable[0x7] == (uintptr_t)_ccm_segment_7SegmentRomStart)) {
```
Comment (thecozies) [inline src/game/texscroll.c:120]: looks like your level is double scrolling now, can you remove this second one?

File: src/game/texscroll/ccm_texscroll.inc.c [hunk 2]
```diff
@@ -9,4 +9,5 @@ void scroll_textures_ccm() {
 	scroll_ccm_level_geo_tank_base();
 	scroll_ccm();
 	scroll_ccm_level_geo_rmine();
+	scroll_ccm();
```
Comment (thecozies) [inline src/game/texscroll/ccm_texscroll.inc.c:12]: actually your level is sextuple scrolling now monkaS

**Review by Cheezepin — APPROVED** [review]

File: src/game/texscroll/ccm_texscroll.inc.c (hunk 2 above)
Comment (Rovert64) [reply src/game/texscroll/ccm_texscroll.inc.c:12]: wouldn't this be triple scrolling

Comment (Rovert64) [reply src/game/texscroll/ccm_texscroll.inc.c:12]: anyway it fixed

**Pull request merged**